compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1296,18 +1300,26 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
              return pybind11::bytes(
                  clientParametersSerialize(clientParameters));
            })
+      .def("lwe_secret_key_param_at",
```

Comment:
naming? is not actually at (like a gate position bellow) maybe just removing at is a better naming?